index.html

```diff
@@ -444,96 +440,94 @@
                However in Polymer 1.2 there is no dynamic type injection for
                DOM stamping... oh well.
               -->
-
-
+          <template is="dom-if" class="tpl learn-more" if="[[showPage('learn-more', tab)]]">
```

Comment:
@jketreno this i am doing, adding some selectors on the template tag to identify the pages regardless the content inside the conditional template does not exist yet or is not loaded.